### PR TITLE
Add more pages

### PR DESCRIPTION
**Pull request by cloudsInWater**
**What this PR does / why we need it**:

1、blog内容先不放
2、community 链接修改
3、页面细节调整

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# cyclone-website
\ No newline at end of file
+# cyclone-website
```
Comment (supereagle) [inline README.md:1]: Please translate to English.

**Comment by cloudsInWater** [comment]
已提交英文说明的 README.md

**Pull request merged**